tests/testthat/test-module_teal.R

```diff
@@ -1,5 +1,10 @@
 # comment: srv_teal is exported so the tests here are extensive and cover srv_data as well.
 #          testing of srv_data is not needed.
+
+
+# utils -----------------------------------------------------------------------------------------------------------
+
+
```

Comment:
This isn't needed. Tests are grouped within `testthat::describe/it`. One can easily fold/unfold whole sections

Reply:
This is not for tests, this is for the part that happens before tests

<img width="629" alt="image" src="https://github.com/user-attachments/assets/be81336f-1c66-4b3b-9d61-66ef014c1247">

Reply:
Will remove, since I removed decorators section, then this is not needed.